### DISCUSSION TITLE
DuckDB CLI on Windows requires VCRUNTIME?

### DISCUSSION THREAD

**Discussion by eitsupi** [duckdb/duckdb#9722] [Q&A] [answered]
I thought DuckDB CLI had no dependencies, but maybe that's not the case on Windows?
When `duckdb.exe` is executed on Windows Sandbox, the following error occurs and it cannot be executed.
https://github.com/microsoft/winget-pkgs/pull/126952#issuecomment-1816919398

**Comment by l1t1** [comment]
yes, it requires. when I run it in a new installed Windows 10, it raise the error.

**Comment by Mause** [comment, marked as answer]
Yeah it's in our list of things we need to document: https://github.com/duckdb/duckdb-web/issues/1301

**Reply by eitsupi** [reply]
I see, thanks!
